io-sim/src/Control/Monad/IOSim/Types.hs

```diff
@@ -633,7 +634,15 @@ instance MonadAsync (IOSim s) where
   asyncWithUnmask k = async (k unblock)
   asyncOnWithUnmask _ k = async (k unblock)
 
+-- | @since 1.4.1.0 This provides access to (almost) everything from the
+-- @primitive@ package, but don't try to use the @MVar@s as that will not
+-- work as expected.
```

Comment:
```suggestion
-- | This provides access to (almost) everything from the
-- @primitive@ package, but don't try to use the @MVar@s as that will not
-- work as expected.
--
-- @since 1.4.1.0
```